rtl/eclrtl/rtlrecord.hpp

```diff
@@ -148,4 +318,92 @@ class ECLRTL_API RtlRecordSize : public IRecordSize, public RtlCInterface
     RtlRecord offsetInformation;
 };
 
+class CSourceRowPrefetcher;
+
+class ECLRTL_API COutputMetaData : implements IOutputMetaData, public RtlCInterface
+{
+public:
+    RTLIMPLEMENT_IINTERFACE
+    COutputMetaData() { recordAccessor[0] = recordAccessor[1] = NULL; }
+    ~COutputMetaData() { delete recordAccessor[0]; delete recordAccessor[1]; }
```

Comment:
Possibly better out of line - no need for these functions to be inline any more.